modules/cli/src/main/java/org/apache/ignite/cli/builtins/config/ConfigurationClient.java

```diff
@@ -69,21 +69,24 @@ public ConfigurationClient(HttpClient httpClient) {
      * @param host String representation of server node host.
      * @param port Host REST port.
      * @param rawHoconPath HOCON dot-delimited path of requested configuration.
+     * @param type Configuration type: {@code node} or {@code cluster}.
+     *      TODO: Fix in https://issues.apache.org/jira/browse/IGNITE-15285
```

Comment:
It this a valid TODO syntax? Can IDEA recognize it? I would suggest creating additional single-line comment instead